Dizzy/Screens/PlaceStory/PlaceStoryVM.swift

```diff
@@ -152,3 +155,13 @@ extension PlaceStoryVM: StoriesInteractorDelegate {
         })
     }
 }
+
+extension PlaceStoryVM: PlacesInteractorDelegate {
```

Comment:
same as my prev comments,
no need for delegate to `PlacesInteractor`

Reply:
Done